middleware/src/main/java/br/ufrn/dimap/middleware/lifecycle/PerRequestLifeCycleManagerI.java

```diff
@@ -0,0 +1,39 @@
+package br.ufrn.dimap.middleware.lifecycle;
+
+import java.util.UUID;
+
+import br.ufrn.dimap.middleware.identification.ObjectId;
+import br.ufrn.dimap.middleware.remotting.interfaces.Invoker;
+
+/**
+ * Implements the Lifecycle manager for the per-request pattern in the middleware.
+ * This service will manage invocations from the Invoker and access/create pools of objects when
+ * necessary.
+ * 
+ * @author Gabriel Victor de Assis Azevedo 
+ * @version 1.0
+ */
+public interface PerRequestLifeCycleManagerI {
```

Comment:
Consider to use the same pattern for names as the other classes and interfaces in this project, which is "interface Name" - instead of "interface NameI" - and  "class MoreSpecificName" (or "class NameImpl").